components/youtube_video/youtube_video.tsx

```diff
@@ -68,18 +71,17 @@ export default class YoutubeVideo extends React.PureComponent {
                     src={'https://www.youtube.com/embed/' + videoId + '?autoplay=1&autohide=1&border=0&wmode=opaque&fs=1&enablejsapi=1' + time}
                     width='480px'
                     height='360px'
-                    type='text/html'
```

Comment:
Looks like that was a mistake in whatever old code sample we based the YouTube preview off of: https://stackoverflow.com/a/4810144